discourse/janitor.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Discourse",
+  "description": "Discourse is the 100% open source discussion platform built for the next decade of the Internet.",
+  "icon": "https://janitor.technology/img/discourse.png",
```

Comment:
Nit: This image doesn't exist yet. Also I'd probably give you a hard time accepting it into Janitor because it's a `png` and not an `svg`. 😋 Then again you could reply that this `"icon"` field is not used anywhere right now.

Reply:
I couldn't find an SVG version of the Discourse icon. We could ask on Meta?

Reply:
Hm, I agree it's probably a good idea to ask on Meta.

Searching for "filetype:svg discourse" on Google Images, I found https://superbig.co/uploads/images/icons/discourse-sso/icon.svg and https://linkerd.io/images/logo_discourse.svg but they don't look super clean so it would be nice to have a proper logo.

P.S. https://www.datocms-assets.com/2885/1508272714-discourse_logo.svg has a logo that looks crisp, so we could also hack this SVG to remove the extra letters.

Reply:
Actually I did what my P.S. says in https://github.com/JanitorTechnology/janitor/pull/190 But maybe this logo is still worth sharing on Meta, since they don't seem to have a clean one?